client/blocks/inline-help/index.jsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { identity } from 'lodash';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import debugFactory from 'debug';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal Dependencies
+ */
+import config from 'config';
+import { tracks } from 'lib/analytics';
+import getGlobalKeyboardShortcuts from 'lib/keyboard-shortcuts/global';
+import Button from 'components/button';
+import Popover from 'components/popover';
+import InlineHelpSearchResults from './inline-help-search-results';
+import InlineHelpSearchCard from './inline-help-search-card';
+import { getInlineHelpSearchResultsForQuery, getSearchQuery } from 'state/inline-help/selectors';
+
+/**
+ * Module variables
+ */
+const globalKeyBoardShortcutsEnabled = config.isEnabled( 'keyboard-shortcuts' );
+const globalKeyboardShortcuts = globalKeyBoardShortcutsEnabled ? getGlobalKeyboardShortcuts() : null;
+const debug = debugFactory( 'calypso:inline-help' );
+
+class InlineHelp extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	state = {
+		showInlineHelp: false,
+	};
+
+	componentDidMount() {
+		if ( globalKeyboardShortcuts ) {
+			globalKeyboardShortcuts.showInlineHelp = this.showInlineHelp.bind( this );
+		}
+	}
+
+	toggleInlineHelp = () => {
+		const { showInlineHelp } = this.state;
+		if ( showInlineHelp ) {
+			this.closeInlineHelp();
+		} else {
+			this.showInlineHelp();
+		}
+	};
+
+	showInlineHelp = () => {
+		debug( 'showing inline help.' );
+		tracks.recordEvent( 'calypso_inline-help_show' );
```

Comment:
I don't think that Tracks [event names](https://github.com/Automattic/wp-calypso/blob/master/client/lib/analytics/README.md#tracks-naming-conventions) support dashes.